spec/mailers/notify_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)